cli/scancommands.go

```diff
@@ -431,7 +440,7 @@ func CurationCmd(c *components.Context) error {
 	if err != nil {
 		return err
 	}
-	threads, err := curation.DetectNumOfThreads(threadsFlag)
+	threads, err := commands.DetectNumOfThreads(threadsFlag)
 	if err != nil {
 		return err
 	}
```

Comment:
```suggestion
	threads, err := pluginsCommon.GetThreadsCount(c)
```
Use already existing func, with exiting general flag